### PR TITLE
[INCR-93] Run python-modernize on crowdsourcehinter

### PR DESCRIPTION
**Pull request by cclauss**
Fixes: https://openedx.atlassian.net/browse/INCR-93

$ __python-modernize -w .__

### CODE REVIEW

File: crowdsourcehinter/crowdsourcehinter.py [hunk 1]
```diff
@@ -1,19 +1,20 @@
...
 import logging
 import operator
 import pkg_resources
 import random
 import json
 import copy
 
-import urllib
-import HTMLParser
+import six.moves.urllib.request, six.moves.urllib.parse, six.moves.urllib.error
```
Comment (jmbowman) [inline crowdsourcehinter/crowdsourcehinter.py:9]: The `request` and `error` imports seem unused, they can probably be dropped.  I'd be inclined to use `from six.moves.urllib.parse import unquote_plus` instead to make the code below marginally cleaner, but it probably doesn't matter that much.

**Comment by cclauss** [comment]
Perhaps we should add [__flake8 —select=F4 .__](http://flake8.pycqa.org/en/latest/user/error-codes.html) or the pylint equivalent to our quality checks to systematically flag unused imports.

**Comment by jmbowman** [comment]
That's one of the pylint checks that edx-lint enables by default, but this XBlock isn't developed by edX (although we use it) and hasn't started to use a linter yet.  I thought we could propose fixing known issues under Python 3 first and then ask @solashirai about what type of linting and/or CI would be welcome.

**Comment by estute** [comment]
@solashirai just wanted to keep this on your radar. Do you have time to review this? Would you be opposed to running this on Travis CI?

**Comment by solashirai** [comment]
@estute My apologies for the delay. Since I haven't kept up on this whatsoever for the past few years, I'm perfectly fine with any changes/running things that you would like.

**Comment by estute** [comment]
@solashirai There are going to be more PRs coming for this repo, and we will need to expose the repo to Travis. Rather than have you be on the hook for this repo if you aren't actively following it, we can fork it into our organization. We will need to license it, most likely with the GNU Affero General Public License, version 3. Do you have any issues with this?

**Comment by solashirai** [comment]
@estute I have no issues with that at all. Would you be able to point me towards what steps I need to take?

**Comment by estute** [comment]
@solashirai you don't need to do anything else. I'll fork the repo into edx, add a License and take from there. Thank you very much!

**Pull request merged**